apps/src/gamelab/GameLab.js

```diff
@@ -840,7 +836,12 @@ GameLab.prototype.runButtonClick = function() {
 /**
  * Execute the user's code.  Heaven help us...
  */
-GameLab.prototype.execute = function(keepTicking = true) {
+GameLab.prototype.execute = function(forPreview = false) {
```

Comment:
I wonder if there is a middle ground with something more descriptive than `keepTicking`, but more abstract and implementation-agnostic than `forPreview`. Ex: 'looping', 'continueLooping', 'singleFrame'?
Adding documentation of the param values might be a way to make the purpose of the parameter clearer.